omero/developers/Python.rst

```diff
@@ -375,6 +431,28 @@ Read-Annotate groups, this will include other users' data - see
     image = conn.getObject("Image", imageId)
     print "Image: ", image,
 
+- **To save an Object's owner as another user (Admins only)**
+
+::
+
+    tag_ann = omero.gateway.TagAnnotationWrapper(conn)
+    tag_ann.setTextValue("Not owned by me")
+    # update details of the wrapped omero.model.AnnotationI _obj
```

Comment:
I would guess that the whole "wrapped objects" workflow and syntax is explained somewhere else in this doc or another doc ? (the underscore with a ... space ?) Definitely a point which warrants more attention I guess.

Reply:
I think this is explained in a bit more detail above and further at https://docs.openmicroscopy.org/omero/5.5.1/developers/PythonBlitzGateway.html